TWLight/users/templates/users/my_applications.html

```diff
@@ -1,91 +1,95 @@
-{% extends "base.html" %}
+{% extends "new_base.html" %}
 {% load i18n %}
 {% load l10n %}
 
 {% block content %}
-  {% comment %}Translators: Heading of the applications page where users can view all of their applications listed with other relevant info.{% endcomment %}
-  <h3 class="inline-header">{% trans 'My applications' %}</h3>
-    <a href="{% url 'users:my_library' %}" class="btn btn-default pull-right">
-    {% comment %}Translators: A button on the 'your applications' page which when clicked takes users their collection page.{% endcomment %}
-    {% trans "My Library" %}
-  </a>
-  <div class="clearfix"></div>
-  {% for app in object_list %}
-    <div class="row">
-      <div class="col-sm-2 hidden-xs">
-        <span class="label label{{ app.get_bootstrap_class }} pull-right">
-          {% if app.parent %}
-            Renewal -
-          {% endif %}
-          {{ app.get_status_display }}
-        </span>
-      </div>
-      <div class="col-xs-8 col-sm-6">
-        <h4>
-        <a href="{{ app.get_absolute_url }}">
-          {{ app.partner }}
-        </a>
-        </h4>
-        <div class="visible-xs">
-          <span class="label label{{ app.get_bootstrap_class }}">
+  {% include "header_partial_b4.html" %}
+  <div id="main-content">
+    {% include "message_partial.html" %}
```

Comment:
nitpick: This partial is outside the main-content div in all other cases. Having it here doesn't break anything on the page, but it does result in the messages having a different position than all the other pages that use the partial.